### PR TITLE
♻️ Storybook in Component Story Format

### PR DESCRIPTION
**Pull request by alanorozco**
From [Storybook's docs](https://github.com/storybookjs/storybook/blob/next/lib/core/docs/storiesOf.md):

> In Storybook 5.2 we introduced a simpler and more portable [Component Story Format](https://storybook.js.org/docs/react/api/csf), and all future tools and infrastructure will be oriented towards CSF. Therefore, we recommend migrating your stories out of `storiesOf` API, and even provide [automated tools to do this](https://github.com/storybookjs/storybook/blob/next/lib/core/docs/storiesOf.md#component-story-format-migration).

AMP samples were using `storiesOf()`, but Preact samples already used Component Story Format.

Used migration tool mentioned.

### CODE REVIEW

File: extensions/amp-animation/0.1/storybook/Basic.amp.js [hunk 1]
```diff
@@ -50,30 +49,38 @@ const BLOCK_STYLE = {
...
+export default {
+  title: 'Animation',
+  decorators: [withKnobs, withAmp],
+
+  parameters: {
+    extensions: [{name: 'amp-animation', version: 0.1}],
+  },
+};
+
+export const Default = () => {
```
Comment (dvoytenko) [inline extensions/amp-animation/0.1/storybook/Basic.amp.js:61]: Could you pls check if the `_default` name is recommended in these cases?

**Review by dvoytenko — APPROVED** [review]

File: extensions/amp-animation/0.1/storybook/Basic.amp.js (hunk 1 above)
Comment (alanorozco) [reply extensions/amp-animation/0.1/storybook/Basic.amp.js:61]: (As in other thread)

Could not find `_default` in their docs, which would contradict this:

> [**Named story exports**](https://storybook.js.org/docs/react/api/csf#named-story-exports)
> ...
> It's recommended to start export names with a capital letter.

But if we prefer `_default` I'd be happy to make the change. We can consider `_Default` too.

Comment (alanorozco) [reply extensions/amp-animation/0.1/storybook/Basic.amp.js:61]: Oh, and this is how the provided codemod modified that name automatically.

Comment (dvoytenko) [reply extensions/amp-animation/0.1/storybook/Basic.amp.js:61]: Let's keep as you have then.

**Review by krdwan — COMMENTED** [review]
LGTM, thanks for doing this!

**Review by krdwan — COMMENTED** [review]
LGTM, thanks for doing this!

**Pull request merged**